.travis.yml

```diff
@@ -9,6 +9,7 @@ branches:
   except:
     - piptools-ignore-patch
 install:
+  - "python -m pip install pip --upgrade"
```

Comment:
If I'm right, the `pip` that we have access to within Travis is in a virtual environment. So in that case, `pip` is always specific to the Python version in use. Correct?